drain-service/drain_training_inferencing.py

```diff
@@ -94,8 +102,11 @@ async def persist_model(batch_processed_queue, current_template_miner):
 async def reset_model(model_training_signal_queue, current_template_miner):
     while True:
         payload = await model_training_signal_queue.get()
-        current_template_miner.reset_model()
-        current_template_miner.save_state()
+        if payload is None:
+            break
```

Comment:
I think this should be `continue` instead of `break`?

Reply:
yes, fixing that.